db/sequence_allocator.go

```diff
@@ -34,6 +34,9 @@ const (
 
 	// Idle batch size.  Initial batch size if SG is in an idle state (with respect to writes)
 	idleBatchSize = 1
+
+	//minimum number that _sync:seq will be corrected by in event of a rollback
```

Comment:
Might be worth adding the commentary about how we settled on this number for future reference.

```suggestion
	// minimum number that _sync:seq will be corrected by in event of a rollback
	// assumes a SG cluster size of 50 nodes each allocating a full batch size of 10
	// if this value is too low and this correction has potential to allocate sequences that other nodes have already reserved a batch for
```